commit 959973b92d3ba235edfa5dcb5df1be1e5d1deba2
Author: Takashi Iwai <tiwai@suse.de>
Date:   Wed Nov 5 11:30:56 2008 +0100

    ALSA: hda - Add a quirk for MEDION MD96630
    
    Use model=lenovo-ms7195-dig for MEDION MD96630 laptop (17c0:4085)
    with ALC888 codec.
    Reference: Novell bnc#412548
    	https://bugzilla.novell.com/show_bug.cgi?id=412528
    
    Signed-off-by: Takashi Iwai <tiwai@suse.de>

Adjusted to apply to Debian's 2.6.26 by dann frazier <dannf@debian.org>

diff -urpN linux-source-2.6.26.orig/sound/pci/hda/patch_realtek.c linux-source-2.6.26/sound/pci/hda/patch_realtek.c
--- linux-source-2.6.26.orig/sound/pci/hda/patch_realtek.c	2009-02-08 23:24:29.000000000 -0700
+++ linux-source-2.6.26/sound/pci/hda/patch_realtek.c	2009-02-08 23:41:17.000000000 -0700
@@ -7879,6 +7879,7 @@ static struct snd_pci_quirk alc883_cfg_t
 	SND_PCI_QUIRK(0x17aa, 0x3bfc, "Lenovo NB0763", ALC883_LENOVO_NB0763),
 	SND_PCI_QUIRK(0x17aa, 0x3bfd, "Lenovo NB0763", ALC883_LENOVO_NB0763),
 	SND_PCI_QUIRK(0x17c0, 0x4071, "MEDION MD2", ALC883_MEDION_MD2),
+	SND_PCI_QUIRK(0x17c0, 0x4085, "MEDION MD96630", ALC888_LENOVO_MS7195_DIG),
 	SND_PCI_QUIRK(0x17f2, 0x5000, "Albatron KI690-AM2", ALC883_6ST_DIG),
 	SND_PCI_QUIRK(0x1991, 0x5625, "Haier W66", ALC883_HAIER_W66),
 	SND_PCI_QUIRK(0x8086, 0x0001, "DG33BUC", ALC883_3ST_6ch_INTEL),
